src/components/Toast/ToastNotice.tsx

```diff
@@ -0,0 +1,46 @@
+import { ReactNode } from "react";
+import { Icon, IconKey } from "src/components/Icon";
+import { IconButton } from "src/components/IconButton";
+import { Css, Palette } from "src/Css";
+import { useTestIds } from "src/utils";
+
+export type ToastNoticeTypes = "error" | "warning" | "success" | "info" | "alert";
+export interface ToastNoticeProps {
+  type: ToastNoticeTypes;
+  message: ReactNode;
+  /** Removes the snackbar notice from the stack */
+  onClose: () => void;
+}
+
+export function ToastNotice(props: ToastNoticeProps) {
```

Comment:
Question/suggestion: Just curious, but why is `ToastNotice` extracted/separate from `Toast`?

Like could we put all of these props/JSX directly into `Toast`? And have a `ToastType` instead of a `ToastNoticeType`?